lib/standalone-single-spa.js

```diff
@@ -18,6 +18,8 @@ const defaultOptions = {
   importMap: { imports: {} },
   disabled: false,
   HtmlWebpackPlugin,
+  importMapOverrides: true,
```

Comment:
Making the default `true` is the breaking change here, so I'll publish a major. I think it's the right default.